doc/classes/File.xml

```diff
@@ -46,42 +46,42 @@
 			<argument index="0" name="path" type="String">
 			</argument>
 			<description>
-				Get whether or not the file in the specified path exists.
+				Gets whether or not the file in the specified path exists.
```

Comment:
why not returns instead of gets

Reply:
Yes, was here, just fixed to Gets. 
Will be in a second pass and rewrite this to "Returns [code]true[/code] if the...."